src/main/java/com/ita/if103java/ims/dto/ForgotPasswordDto.java

```diff
@@ -0,0 +1,34 @@
+package com.ita.if103java.ims.dto;
+
+import com.ita.if103java.ims.dto.transfer.NewData;
+
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+
+public class ForgotPasswordDto implements Serializable {
+    @NotBlank(groups = {NewData.class},
+        message = "Please, enter these data")
+    private String email;
```

Comment:
Why do you need to wrap single String field into object? For validation only?

Reply:
Yes, and I think it is better to get data in  JSON from UI